### PR TITLE
feat: Empty fallback results tabs

### PR DESCRIPTION
**Pull request by rajsam003**
This PR resolves [GRO-1533] 

### Description

Tabs content are the part of these tickets
- https://artsyproduct.atlassian.net/browse/GRO-1537
- https://artsyproduct.atlassian.net/browse/GRO-1538

image

### PR Checklist

- [x] I tested my changes on **iOS** / **Android**.
- [x] I added screenshots or videos to illustrate my changes.
- [ ] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [x] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [x] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[GRO-1533]: https://artsyproduct.atlassian.net/browse/GRO-1533?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by TMcMeans** [comment]
**question [non-blocking]:** 

has there been any copy text approved by design for empty states? Please ignore if this was discussed previously or if it will get updated later :) 

*format via [Conventional Comments](https://conventionalcomments.org/)*

File: src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResults.tsx [hunk 1]
```diff
@@ -1,106 +1,36 @@
...
-          </Flex>
-          <Text variant="lg-display">Art Taste Quiz</Text>
-          <Text color="black60">Results Completed</Text>
-        </Flex>
-      </Screen.Body>
-    </Screen>
-  )
-}
+  if (!isResultReady) {
+    return <ArtQuizResultsLoader isReady />
```
Comment (laurabeth) [inline src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResults.tsx:25]: Design pointed out that we should not show this loader if the user is returning to their results. We should lift this into the ArtQuizArtworks view to be displayed after the last artwork. 

Is this complicated by the use of suspense?

File: src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizResultsEmptyTabs.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+              subtitle="There are almost 2 million artworks on Artsy—keep exploring to find something you love."
+            />
+          }
+        />
+      </Screen.Body>
+    </Screen>
+  )
+}
+
+const EmptyScreen = () => {
```
Comment (TMcMeans) [inline src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizResultsEmptyTabs.tsx:41]: **question [blocking]:** 

Just checked the designs and realized that the backfill isn't going to be just copy text, but an artwork grid. Is that to be included in a follow up PR?

*format via [Conventional Comments](https://conventionalcomments.org/)*

File: src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizResultsTabsHeader.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+export const ArtQuizResultsTabsHeader = ({ title, subtitle }: ArtQuizResultsTabsHeaderProps) => {
+  return (
+    <Flex px={2}>
+      <Text variant="lg">{title}</Text>
+      <Text variant="sm" color="black60">
+        {subtitle}
+      </Text>
+      <Spacer m={1} />
+      <Button size="small" variant="outlineGray">
+        Email My Results
```
Comment (laurabeth) [inline src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizResultsTabsHeader.tsx:17]: Just checked force, looks like this button shouldn't appear on the fallback page.

**Review by TMcMeans — COMMENTED** [review]
I think it makes sense to add the query & UI for the trending artists/artworks backfill grid to this PR since it's the default state.

**Review by laurabeth — COMMENTED** [review]
Code looks great! 

Did some comparisons w/ the figma and saw 2 minor things. Once we bring those in line, let's merge.

File: src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizResultsEmptyTabs.tsx (hunk 2 above)
Comment (rajsam003) [reply src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizResultsEmptyTabs.tsx:41]: Yes , Here are the tickets to work on that. 
https://artsyproduct.atlassian.net/browse/GRO-1537
https://artsyproduct.atlassian.net/browse/GRO-1538

File: src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResults.tsx (hunk 1 above)
Comment (rajsam003) [reply src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResults.tsx:25]: Hmmm.. but how would this be any different? I mean the loader will get displayed after liking/disliking the last artwork before the result screen. So we can put this loader either in `<ArtQuizArtworks\>` or `<ArtQuizResults\>`.  I dont see any difference in that.

File: src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizResultsTabsHeader.tsx (hunk 3 above)
Comment (rajsam003) [reply src/app/Scenes/ArtQuiz/ArtQuizResults/ArtQuizResultsTabs/ArtQuizResultsTabsHeader.tsx:17]: Does it appear on the fallback page? It is supposed to be in the Result screen Header, right?

**Pull request merged**